motoko/README.md

```diff
@@ -12,6 +12,7 @@ These examples show the Motoko language as a backend language for the IC, operat
 - [`factorial`](https://github.com/dfinity/examples/tree/master/motoko/factorial) -- basic factorial demo.
 - [`hello-world`](https://github.com/dfinity/examples/tree/master/motoko/hello-world) -- basic hello world demo.
 - [`hello_cycles`](https://github.com/dfinity/examples/tree/master/motoko/hello_cycles) -- basic cycles demo.
+- [`persistant-storage`](https://github.com/dfinity/examples/tree/master/motoko/persistant-storage) -- basic stable var demo.
```

Comment:
```suggestion
- [`persistent-storage`](https://github.com/dfinity/examples/tree/master/motoko/persistant-storage) -- basic stable var demo.
```

Reply:
I would rename the folder to` persistent-storage` too.

Reply:
the URL has the same typo